tdisplay-s3-i2c-ano.yaml

```diff
@@ -0,0 +1,409 @@
+
```

Comment:
instead of copying this to a new file, just add the include for the _seesaw.yaml in the tdisplay-s3.yaml and comment out the other one